blog.html

```diff
@@ -14,13 +20,122 @@
 	</head>
 	<body>
 		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+			<header class="clearfix">
+				<section="logo">
+				<h1>
+					Relaxr
+				</h1>
+			</section>
+			<!-- <nav class="clearfix"> -->
+				<ul class="top-nav clearfix">
+					<li><a href="#"><span class="nav">Blog</span></a></li>
+					<li><a href="#"><span class="nav">Contact Us</span></a></li>
+					<li><a href="#"><span class="nav">Team</span></a></li>
+					<li><a href="#"><span class="nav">FAQ</span></a></li>
+					<li><a href="#"><span class="nav">About</span></a></li>
+				</ul>
+			<!-- </nav> -->
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+
+
+		<main class="clearfix">
+			<div class="articles clearfix">
+			<article>
+				<div class="article-header">
```

Comment:
You could even use a `<header>` element here. Sounds like it is semantically appropriate from the name of your class. Important to remember a `<header>` isn't only for us on the header of your entire page. It could also be for the header of an article. Same goes for `<footer>` as well.